src/ray/raylet/node_manager.cc

```diff
@@ -3007,6 +3007,16 @@ MemoryUsageRefreshCallback NodeManager::CreateMemoryUsageRefreshCallback() {
             ray::stats::STATS_memory_manager_worker_eviction_total.Record(
                 1, "MemoryManager.ActorEviction.Total");
           }
+        } else {
```

Comment:
nit: it's hard to track the start if/else branch for this block. consider refactor it into its own helper function/class?